libscript/src/script-module.cpp

```diff
@@ -983,78 +1014,115 @@ void MCScriptReleaseRawModule(MCScriptModule *p_module)
 
 ////////////////////////////////////////////////////////////////////////////////
 
-bool MCScriptLookupPropertyDefinitionInModule(MCScriptModuleRef self, MCNameRef p_property, MCScriptPropertyDefinition*& r_definition)
+static bool
+__MCScriptLookupDefinitionInModule(MCScriptModuleRef self,
+								   MCScriptDefinitionKind p_kind,
+								   MCNameRef p_name,
+								   MCScriptDefinition*& r_definition)
```

Comment:
In order to avoid an awkward cast in all the callers, I wonder if this could be written as:

```C++
template <typename T, MCScriptDefinitionKind K>
__MCScriptLookupDefinitionInModule(MCScriptModuleRef self, MCNameRef p_name, T*& r_definition)
{
    /* ... */
}
```